docs/reference-implementation/Getting-Started.md

```diff
@@ -23,17 +25,12 @@ The following must be installed on the client machine used to deploy Azure Missi
 
 - [Azure CLI](https://docs.microsoft.com/cli/azure/service-page/azure%20cli?view=azure-cli-latest)
```

Comment:
Isn't is ironic that for the "UI" version you still need Azure CLI? :D It's to create the RBAC service principal which is admittedly easier to do in the command line.

Reply:
good callout! I changed this now, how about this?

Reply:
"Create Azure service principal" is still Step 4 in the guide which requires Azure CLI.

https://github.com/Azure/Mission-Critical-Online/blob/8cac31c637eaae793163bc18ab41a005125ede3f/docs/reference-implementation/Getting-Started.md#4-create-azure-service-principal